docs/README.md

```diff
@@ -7,7 +7,7 @@ Homebrew installs the stuff you need that Apple didn’t.
 - [Gems, Eggs and Perl Modules](Gems,-Eggs-and-Perl-Modules.md)
 - [Homebrew and Python](Homebrew-and-Python.md)
 
-Please note: Homebrew is not 1.0 yet. Generally it works well, but when
+Please note: Homebrew generally works well, but when
 it doesn’t we’d hope you report the bug.
```

Comment:
It would be nice if you could also add a new line after the paragraph, thanks. And refill the lines to 70 columns maybe.